pywps/inout/outputs.py

```diff
@@ -201,7 +202,10 @@ def _json_reference(self, data):
         if self.prop == 'url':
             data["href"] = self.url
         elif self.prop is not None:
-            self.storage = FileStorageBuilder().build()
+            if self.data_format.mime_type == "application/x-ogc-dods":
```

Comment:
This does not check the configured storagetype ... whenever I have a `dods` file it will use the `DapStorageBuilder` even when it is not configured.

Reply:
Indeed.